guides/common/modules/proc_synchronizing-the-satellite-tools-repository.adoc

```diff
@@ -1,26 +1,57 @@
 [id="Synchronizing_the_Client_Repository_{context}"]
 = Synchronizing the {project-client-name} Repository
 
-ifdef::satellite[]
 Use this section to synchronize the {project-client-name} repository from the Red Hat Content Delivery Network (CDN) to your {Project}.
 This repository provides the `katello-agent`, `katello-host-tools`, and `puppet` packages for clients registered to {ProjectServer}.
-endif::[]
+Select the applicable operating system and version to synchronize the client repository.
+
+* xref:#synchronizing-repos-rhel-9[{RHEL} 9]
+* xref:#synchronizing-repos-rhel-8[{RHEL} 8]
+
+== [[synchronizing-repos-rhel-9]]{RHEL} 9
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Synchronizing_the_Client_Repository_rhel_9{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
++
+A list of product repositories available for synchronization is displayed.
```

Comment:
```suggestion
```

We generally don't document what is shown at each page.

If this is important, maybe append "to see product repositories available for synchronization" to line 16.